docs/start/install.rst

```diff
@@ -170,6 +170,21 @@ Step 6: do a rollout!
 
 Now you should have a working Shipper installation. :ref:`Let's roll something out! <user_rolling-out>`
 
+
+*****************
+Namespace manager
+*****************
+By design, Shipper does not create namespaces in the application cluster.
+In case the namespace does not exist in the application cluster, and this application cluster is selected for a *Release*,
```

Comment:
Should we specify here just for completeness that the expected namespace in the application cluster has to be of the same name?

Reply:
hmm you're right